pkg/services/ldap/settings.go

```diff
@@ -135,9 +138,28 @@ func readConfig(configFile string) (*Config, error) {
 		}
 	}
 
+	for _, server := range result.Servers {
+		envValue := evalEnvVar(server.BindPassword)
+		server.BindPassword = envValue
+	}
+
 	return result, nil
 }
 
+func evalEnvVar(value string) string {
+	regex := regexp.MustCompile(`\${(\w+)}`)
+	if regex != nil {
```

Comment:
We shouldn't need the nil check. Add a test instead. If the regexp compiles in test it will never be nill.